### ISSUE TITLE
Section 508 accessibility test run

### ISSUE THREAD

**Issue by madelyngreyes** [ImagingDataCommons/IDC-WebApp#515]
Completed a section 508 accessibility test on the IDC production tier using the [Wave](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh) google extension tool. 

All pages had errors or contrast errors except the user details page.  

A detailed report of the test run can be found here, [Section 508 - Dec 2020 run](https://docs.google.com/presentation/d/19srSHANflsQP-A4mfuHl2fg2WL6d9bw_LQhLrhpK-EU/edit#slide=id.p).

**Comment by pieper** [comment]
This looks interesting @madelyngreyes - can you let me know the three errors reported for the viewer?

**Comment by madelyngreyes** [comment]
I have taken a screenshot of the specific errors from the viewer below, 

![3 errors from the viewer](https://user-images.githubusercontent.com/16824894/103951599-ffeb4c00-510c-11eb-922f-9a1a7e35bfd6.PNG)

**Comment by pieper** [comment]
Thanks Madelyn - I'm still not sure I understand what it's telling me, but if you think these issues need to be fixed can you file them as [ohif viewer ](https://github.com/OHIF/Viewers/issues) issues and add the IDC:candidate label?

**Comment by madelyngreyes** [comment]
Yes, can do. I have attached the information provided for the missing form label below for reference, 
![mising form label detail](https://user-images.githubusercontent.com/16824894/103953442-2959a700-5110-11eb-8981-166475059d64.PNG)

**Comment by s-paquette** [comment]
We need to separate the OHIF 508 from the WebApp 508 (as we WebApp devs can't work on OHIF UI).
